src/main/java/org/jenkinsci/plugins/github_branch_source/UntrustedPullRequestSCMRevision.java

```diff
@@ -27,14 +27,14 @@
 import jenkins.plugins.git.AbstractGitSCMSource;
 import jenkins.scm.api.SCMHead;
 
-/**
- * Revision of a pull request which should load sensitive files from the base branch.
- */
+@Deprecated
 class UntrustedPullRequestSCMRevision extends AbstractGitSCMSource.SCMRevisionImpl {
     
+    private static final long serialVersionUID = -6961458604178249880L;
+    
     final String baseHash;
 
-    UntrustedPullRequestSCMRevision(SCMHead head, String hash, String baseHash) {
+    private UntrustedPullRequestSCMRevision(SCMHead head, String hash, String baseHash) {
```

Comment:
It was already package-private. Anyway this whole plugin is not exposing any intentional API.